README.md

```diff
@@ -19,7 +19,7 @@ local machine.
 Before the build user can be created, the following profile must exist in
 your AWS credentials file:
 
-* `cool-terraform-backend`
+- `cool-terraform-backend`
```

Comment:
Sorry I missed this on the first pass (and this change isn't a result of this merge) but this role's Terraform code is not designed to be run in the COOL. The `cool-terraform-backend` profile is a bad recommendation as a result.

Reply:
Agreed, and in general I don't like the COOL-specific language here.  I will create an upstream PR to resolve this.

Reply:
See cisagov/skeleton-ansible-role-with-test-user#22.